### ISSUE TITLE
[Enhancement] Routing concept with cookbook example 

### ISSUE THREAD

**Issue by mictro** [kirbydesign/designsystem#2862]
## Describe the enhancement
Add tab routing concept with router-outlet to tab-navigation and amend cookbook with an example. 
A POC on the feasibility of using Angular's native routing has been conducted with participation of @RasmusTraeholt and @mark-drastrup. The use of Angular's native relative-routing and named-outlets a.k.a. auxiliary-routing was tested on the cookbook example but proved not to work as intended. A working solution was chosen based on absolute routing where the absolute route to the navigation tab was provided using a router attribute.
It is to be decided whether the absolute path of the nav-tab router-link should be specified using Angular's routerLink attribute or specified using a custom attribute on the tab-nav component(s).    

## Describe the solution you'd like
- [ ] Add route parameter to the tab-nav-item  and router-outlet to the tab-nav component(s). 
- [ ] Revise the cookbook page tab-nav example to use the router concept. 

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

**Comment by mictro** [comment]
The issue is out of scope for the kirby component feature (see tracking issue #2705 ) - but may become relevant later on as an add-on...
